front-end/components/Dashboard/dashboard.vue

```diff
@@ -1,9 +1,7 @@
 <template>
     <div>
         <div class="dashboard" :class="theme">
```

Comment:
I think this class is not needed anymore

Reply:
Nicely spotted sir

Reply:
It dead